`TestVariableConstantArrayOp` is Flaky

[Jenkins Error](https://jenkins.preferred.jp/job/chainer/job/chainer_pr/2015/TEST=chainer-py35,label=mn1-p100/testReport/junit/tests.chainer_tests.functions_tests.math_tests.test_basic_math/TestVariableConstantArrayOp_param_0_%7Bdtype=float16%7D/test_div_backward_cpu/)

```
RuntimeWarning: Parameterized test failed.  Base test method: TestVariableConstantArrayOp.test_div_backward_cpu Test parameters:   dtype: <class 'numpy.float16'>   (caused by) RuntimeWarning: overflow encountered in true_divide
```

Trace
```self = <chainer.testing._bundle.TestVariableConstantArrayOp_param_0_{dtype=float16} testMethod=test_div_backward_cpu>

    def test_div_backward_cpu(self):
>       self.backward_cpu(lambda x, y: x / y)

self       = <chainer.testing._bundle.TestVariableConstantArrayOp_param_0_{dtype=float16} testMethod=test_div_backward_cpu>

tests/chainer_tests/functions_tests/math_tests/test_basic_math.py:1310: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/chainer_tests/functions_tests/math_tests/test_basic_math.py:1289: in backward_cpu
    self.check_backward(op, self.x, self.gy, lambda x: x, positive)
tests/chainer_tests/functions_tests/math_tests/test_basic_math.py:1286: in check_backward
    dtype=numpy.float64, **options)
chainer/gradient_check.py:900: in check_backward
    detect_nondifferentiable, is_immutable_params=False
chainer/gradient_check.py:464: in run
    self._run()
chainer/gradient_check.py:501: in _run
    xs_backward, ys, params_backward, directions)
chainer/gradient_check.py:616: in _directional_backward_gradients
    chainer.backward(ys, gys)
chainer/_backprop.py:143: in backward
    _backprop_to_all(outputs, False, None)
chainer/_backprop.py:226: in _backprop_to_all
    func, target_input_indexes, out_grad, in_grad, is_debug)
chainer/_backprop_utils.py:140: in backprop_step
    _reraise_with_stack(func, e)
chainer/_backprop_utils.py:138: in backprop_step
    target_input_indexes, grad_outputs)
chainer/functions/math/basic_math.py:421: in backward
    return DivGrad().apply((x[0], x[1], grad_outputs[0]))
chainer/function_node.py:325: in apply
    outputs = self.forward(in_data)
chainer/function_node.py:519: in forward
    return self.forward_cpu(inputs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <chainer.functions.math.basic_math.DivGrad object at 0x7f353f32ba58>
inputs = (array([[ 0.54638672,  0.60302734],
       [ 0.78417969,  0.82080078],
       [ 0.72998047,  0.81347656]], dtype=float...ray([[ 0.73828125, -0.16503906],
       [-0.66064453,  0.66845703],
       [ 0.67871094, -0.2980957 ]], dtype=float16))

    def forward_cpu(self, inputs):
        self.retain_inputs((0, 1, 2))
        x0, x1, gy = inputs
        gx0 = utils.force_array(gy / x1)
>       gx1 = utils.force_array(-gx0 * x0 / x1)
E       RuntimeWarning: Parameterized test failed.
E       
E       Base test method: TestVariableConstantArrayOp.test_div_backward_cpu
E       Test parameters:
E         dtype: <class 'numpy.float16'>
E       
E       
E       (caused by)
E       RuntimeWarning: overflow encountered in true_divide

gx0        = array([[   1.65332031,  155.875     ],
       [   1.44433594,    1.39550781],
       [  -3.35742188,    0.61669922]], dtype=float16)
gy         = array([[ 0.73828125, -0.16503906],
       [-0.66064453,  0.66845703],
       [ 0.67871094, -0.2980957 ]], dtype=float16)
inputs     = (array([[ 0.54638672,  0.60302734],
       [ 0.78417969,  0.82080078],
       [ 0.72998047,  0.81347656]], dtype=float...ray([[ 0.73828125, -0.16503906],
       [-0.66064453,  0.66845703],
       [ 0.67871094, -0.2980957 ]], dtype=float16))
self       = <chainer.functions.math.basic_math.DivGrad object at 0x7f353f32ba58>
x0         = array([[ 0.54638672,  0.60302734],
       [ 0.78417969,  0.82080078],
       [ 0.72998047,  0.81347656]], dtype=float16)
x1         = array([[ 0.4465332 , -0.00105858],
       [-0.45751953,  0.47900391],
       [-0.20214844, -0.48339844]], dtype=float16)

chainer/functions/math/basic_math.py:430: RuntimeWarning
```

--- a/tests/chainer_tests/functions_tests/math_tests/test_basic_math.py
+++ b/tests/chainer_tests/functions_tests/math_tests/test_basic_math.py
@@ -1144,6 +1144,8 @@ class TestVariableConstantArrayOp(unittest.TestCase):
         self.gy = numpy.random.uniform(-1, 1, (3, 2)).astype(self.dtype)
         self.ggx = numpy.random.uniform(.5, 1, (3, 2)).astype(self.dtype)
         self.value = numpy.random.uniform(-1, 1, (3, 2)).astype(self.dtype)
+        # Avoid overflow in div test (especially backward)
+        self.value[abs(self.value) < 1e-2] = 1.
 
     def check_forward(self, op, array_conv, positive):
         value = self.value

